simplesource-command-serialization/src/main/java/io/simplesource/kafka/serialization/avro/AvroSerdes.java

```diff
@@ -207,7 +202,8 @@ private static Schema aggregateUpdateResultSchema(final Schema aggregateSchema)
         private static GenericRecord fromReason(final Schema schema, final CommandError commandError) {
             return new GenericRecordBuilder(schema)
                     .set(ERROR_MESSAGE, commandError.getMessage())
-                    .set(ERROR_CODE, commandError.getReason().name())
+                    .set(ERROR_CLASS, commandError.getClass().getCanonicalName())
```

Comment:
`ERROR_MESSAGE ` and `ERROR_CLASS` are here for debug purpose. They're not used in the deserialization mechanism.